visu3d/dc_arrays/camera_spec.py

```diff
@@ -118,14 +119,60 @@ def w(self) -> int:
 
   @property
   def hw(self) -> tuple[int, int]:
-    """`(Height, Width)` in pixel (for usage in `(i, j)` coordinates)."""
+    """`(Height, Width)` in pixel (for usage in `(u, v)` coordinates)."""
     return (self.h, self.w)
 
   @property
   def wh(self) -> tuple[int, int]:
-    """`(Width, Height)` in pixel (for usage in `(u, v)` coordinates)."""
+    """`(Width, Height)` in pixel (for usage in `(i, j)` coordinates)."""
     return (self.w, self.h)
 
+  @property
+  def fw(self) -> float:
+    """Focal length (along x-axis) in pixels (for usage in `(i, j)` coordinates)."""
+    return self.focal_length[0]
+
+  @property
+  def fh(self) -> float:
+    """Focal length (along y-axis) in pixels (for usage in `(i, j)` coordinates)."""
+    return self.focal_length[1]
+
+  @property
+  def focal_px_wh(self) -> tuple[float, float]:
```

Comment:
This seems redundant with `focal_length` defined line 110.

To keep the API minimal, I would remove this property and rename `focal_length` -> `focal_px_wh` in the dataclass attribute.